website/pages/home/index.jsx

```diff
@@ -112,78 +125,102 @@ export default function HomePage() {
                 url: require('./img/logos/kubernetes-color.svg?url'),
               },
               {
-                url: require('./img/logos/more-gray.svg?url'),
                 alt: 'More integrations',
+                url: require('./img/logos/custom.svg?url'),
               },
             ],
           },
           {
             title: 'Access',
-            description:
-              'Automate credential injection to securely access services and hosts with HashiCorp Vault. Reduce risk of leaking credentials with dynamic secrets and just-in-time credentials.',
-          },
-        ]}
-      />
-
-      <div className="use-cases-section">
-        <div className="g-grid-container">
-          <h2 className="g-type-display-2">Use cases</h2>
-          <UseCases
-            items={[
+            description: (
+              <>
+                Reduce risk of leaking credentials with dynamic secrets and
+                just-in-time credentials. Automate credential injection to
+                securely access services and hosts with{' '}
+                <a href="https://learn.hashicorp.com/tutorials/boundary/vault-cred-brokering-quickstart">
+                  HashiCorp Vault
+                </a>
+                .
+              </>
+            ),
+            logos: [
               {
-                title: 'Easily onboard and manage users',
-                description:
-                  'Use SSO to manage onboarding and off-boarding users.',
-                image: {
-                  url: require('./img/red-usecase-accessmgmt.png?url'),
-                },
-                link: {
-                  title: 'Learn more',
-                  url:
-                    'https://learn.hashicorp.com/tutorials/boundary/getting-started-config',
-                },
+                alt: 'Postgresql',
+                url: require('./img/logos/postgresql.png?url'),
+                width: 36,
               },
               {
-                title: 'Open and extensible remote access',
-                description:
-                  'Integrate with existing tooling and APIs to simplify access.',
-                image: {
-                  url: require('./img/red-usecase-accessprivileges.png?url'),
-                },
-                link: {
-                  title: 'Learn more',
-                  url:
-                    'https://learn.hashicorp.com/tutorials/boundary/manage-users-groups',
-                },
+                alt: 'SSH',
+                url: require('./img/logos/ssh.png?url'),
```

Comment:
Could we export these out at SVGs?

Reply:
Got one https://github.com/hashicorp/boundary/pull/1558/commits/5280899f69e62a077b8734a3d8fa023c88adc50f